aioredis/commands/server.py

```diff
@@ -126,14 +126,26 @@ def debug_segfault(self, key):
         # won't test, this probably works
         return self.execute(b'DEBUG', 'SEGFAULT')  # pragma: no cover
 
-    def flushall(self):
-        """Remove all keys from all databases."""
-        fut = self.execute(b'FLUSHALL')
+    def flushall(self, async=False):
```

Comment:
`async` is a reserved keyword, argument must be renamed (maybe `async_flush` or something better)
https://docs.python.org/3.7/reference/lexical_analysis.html#keywords